label_maker/images.py

```diff
@@ -66,9 +64,11 @@ def class_test(value):
     # download tiles
     tiles = class_tiles + background_tiles
     print('Downloading {} tiles to {}'.format(len(tiles), op.join(dest_folder, 'tiles')))
-    o = urlparse(imagery)
-    _, image_format = op.splitext(o.path)
+
+    # get image acquisition function based on imagery string
+    image_function = download_tile_tms
+    if imagery[-4:].lower() == '.tif':
```

Comment:
Any reason not to use `op.splitext(imagery)[1].lower()` here?

Reply:
Also, could we ever get `.tiff`?

Reply:
No and yes (I was worried about dots in urls but apparently it works fine). I'll test for both extensions